internal/pkg/agent/cmd/install.go

```diff
@@ -249,7 +261,19 @@ func installCmd(streams *cli.IOStreams, cmd *cobra.Command) error {
 		progBar.Describe("Successfully uninstalled Elastic Agent")
 	}
 	if status != install.PackageInstall {
-		ownership, err = install.Install(cfgFile, topPath, unprivileged, log, progBar, streams)
+		customUser, _ := cmd.Flags().GetString(flagInstallCustomUser)
+		customGroup, _ := cmd.Flags().GetString(flagInstallCustomGroup)
+		customPass := ""
+		if runtime.GOOS == "windows" {
+			customPass, _ = cmd.Flags().GetString(flagInstallCustomPass)
+
+			if (customUser != "" || customPass != "") &&
+				(customUser == "" || customPass == "") {
+				return fmt.Errorf("error installing package: all Active Directory parameters must be provided")
```

Comment:
Why? I think this could work by just providing say the user, then still allow the group to be created.

Or maybe you want to use the default user and password, but you want to just set the password with `--password`?

Reply:
Yeah but we need to be vocal about that and raise a warning that we are adding user somewhere or creating group that existing user will be part of